### ISSUE TITLE
ruby -v shows jruby-1.7.13 (its not even installed!) instead of 1.7.16

### ISSUE THREAD

**Issue by Allanon29** [sstephenson/rbenv#654]
I cannot run bundler because I get an error that the bundle command exists only in jruby-1.7.16.
So I check the ruby version with ruby -v, and I get jruby-1.7.13 which is not even installed.
If I type rbenv versions I get:

```
  jruby-1.7.16
* jruby-1.7.16.1
```

The I tried to set the ruby version with rbenv global jruby-1.7.16.1 but ruby -v still shows 1.7.13.
I tried to restart the shell, and even the whole vm, but the same problem exists.

This happened after I updated rbenv recently. Previously it worked fine.
This happened on Centos 6.5

Any ideas how to fix this?

UPDATE:
If I install MRI ruby 2.1.5 and set it as rbenv global then it works. However if I install jruby-1.7.15 and try to set it as global, I get 1.7.13 again with ruby -v

**Comment by Allanon29** [comment]
Hi,

Sorry for the late answer. I tried delete jruby-1.7.16.1. In fact I removed all jruby versions from the system.
Then I get this message:
rbenv: version `jruby-1.7.16' is not installed

When install a jruby version again, the same version problem occurs.

**Comment by Allanon29** [comment]
After a few tries I am sure this problem is related to torquebox because it ships with jruby-1.7.13 inside the torquebox folder, which I usually place inside opt/torquebox/current.
I place my torquebox config in etc/profile.d/torquebox.sh where I set the jruby_home directory. If I set this to one of the jruby-versions in the rbenv directory, then I can change the version to 1.7.16 but TB still tries to use jruby-1.7.13 when I deploy an app.
